packages/fxa-auth-server/lib/routes/utils/signin.js

```diff
@@ -629,7 +629,7 @@ module.exports = (
       // We should only really remove reminders if the session
       // was marked as verified, ie have met all the requirements
       // to start syncing
-      if (response.verified) {
+      if (response.sessionVerified) {
```

Comment:
Does the account's email also need to be verified to start syncing? I'm believe the answer no, but just wanted to double check.

Reply:
No - they'll be put in a "finish account set up state".